test/parallel/test-whatwg-url-properties.js

```diff
@@ -25,6 +25,9 @@ const expected = ['toString',
                   'username', 'password', 'host', 'hostname', 'port',
                   'pathname', 'search', 'searchParams', 'hash', 'toJSON'];
 
+const obj = { toString() { throw new Error('toString'); } };
```

Comment:
`test-whatwg-url-setters` is a WPT-derived file, and therefore doesn't pertain to Node.js-specific things like error messages, for which these tests check.